docs/blog/announcing-vite3.md

```diff
@@ -1,6 +1,9 @@
 ---
-sidebar: false
+title: Vite 3.0 is out!
+author:
+  name: patak
```

Comment:
Let's use "The Vite Team" as author in all the posts after the Vite 2 announcement
```suggestion
  name: The Vite Team
```